.github/workflows/build-dockerimage.yaml

```diff
@@ -12,6 +12,9 @@ on:
       golinkerargs:
         required: true
         type: string
+      cgo_cflags:
+        required: true
```

Comment:
Can we use a default instead of making it required?

Reply:
We could, but it would be inconsistent with `golinkerflags`. I preferred to repeat the pattern.

Reply:
There should be common logic reused in Makefiles and github actions instead of hardcoding it in each place. 
We should rework it in future and imho good starting point is having flags separated from github actions to sh scripts which can be also invoked from Makefile.